src/Protocol/MojangAPI.cpp

```diff
@@ -15,6 +13,19 @@
 #include "../OSSupport/IsThread.h"
 #include "../Root.h"
 
+// Because SQLiteCpp uses NULL instead of nullptr, we need to disable the Clang warning here
+#if __clang_major__ >= 5
+	#pragma clang diagnostic push
+	#pragma clang diagnostic ignored "-Wzero-as-null-pointer-constant"
```

Comment:
This works for now but I think it would be better if the libraries were included as system headers.

Reply:
It would be even better, if we "fix" the SQLiteCpp project (You can detect, if `nullptr` is supported by CMake and inject a header redirecting `nullptr` to `NULL`).

And I'm not the biggest fan of the `isystem` switch. See the explanation on [Reddit](https://www.reddit.com/r/cpp/comments/5arvg9/watch_out_qt_headers_suppress_warnings_with/), why this switch could harm.

Reply:
Thanks for the link, very enlightening.